lib/ace/ext/papeeria/spellchecker.coffee

```diff
@@ -0,0 +1,16 @@
+define (require, exports, module) ->
+    "use strict";
+    # SpellChecker is a class that by now provides only numbers highlighting,
+    # but it will provide a hunspell spellchecking someday.
+    class SpellChecker
```

Comment:
At the moment class is a little bit heavy-weight structure for this purpose. Simple object or even a function would do the trick. It is okay to leave class though, in the hope that it will become more useful afterwards.